static/js/settings_account.js

```diff
@@ -712,11 +712,8 @@ export function set_up() {
     });
 
     $("#show_my_user_profile_modal").on("click", () => {
-        overlays.close_overlay("settings");
         const user = people.get_by_user_id(people.my_current_user_id());
-        setTimeout(() => {
-            user_profile.show_user_profile(user);
-        }, 100);
+        user_profile.show_user_profile(user);
 
         // If user opened the "preview profile" modal from user
         // settings, then closing preview profile modal should
```

Comment:
Shouldn't we delete the `.one` handler just after this as well? I'm pretty sure that was just a workaround for this not working as expected.

(Do we also want `stopPropagation/preventDefault` on this handler?)

Reply:
> Shouldn't we delete the `.one` handler just after this as well? I'm pretty sure that was just a workaround for this not working as expected.
> 
> (Do we also want `stopPropagation/preventDefault` on this handler?)

That was handled by #22570